apps/condo/domains/division/utils/testSchema/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generated by `createschema division.Division 'name:Text; organization:Relationship:Organization:CASCADE; responsible:Relationship:OrganizationEmployee:PROTECT;'`
+ * In most cases you should not change it by hands
+ * Please, don't remove `AUTOGENERATE MARKER`s
+ */
+const faker = require('faker')
+
+const { generateServerUtils, execGqlWithoutAccess } = require('@condo/domains/common/utils/codegeneration/generate.server.utils')
+
+const { generateGQLTestUtils, throwIfError } = require('@condo/domains/common/utils/codegeneration/generate.test.utils')
+
+const { Division: DivisionGQL } = require('@condo/domains/division/gql')
+/* AUTOGENERATE MARKER <IMPORT> */
+
+const Division = generateGQLTestUtils(DivisionGQL)
+/* AUTOGENERATE MARKER <CONST> */
+
+async function createTestDivision (client, organization, responsible, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!organization || !organization.id) throw new Error('no organization.id')
+    if (!responsible || !responsible.id) throw new Error('no responsible.id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    // TODO(codegen): write createTestDivision logic for generate fields
```

Comment:
!

Reply:
create logic is already implemented, removed this comment in `d9810779`